README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только Greenplum 6.
+
+# 2. Компиляция pgBackRest
+
+- Установить зависимости
+
+для CentOS 7:
+```
+yum install gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel lz4-devel libyaml-devel
+```
+
+для ALT Linux p10
+```
+apt-get update
+apt-get install git gcc openssl-devel libxml2-devel bzip2-devel libzstd-devel libyaml-devel zlib-devel 
```

Comment:
these 2 cmds required 'sudo' when I tried these commands on ALT Linux. Better to add it here to the commands.